src/org/daisy/dotify/formatter/impl/search/SheetIdentity.java

```diff
@@ -25,27 +46,15 @@ public SheetIdentity(Space s, Integer volumeIndex, Integer volumeGroup, int shee
         }
         this.volumeIndex = volumeIndex;
         this.volumeGroup = volumeGroup;
-        this.sheetIndex = sheetIndex;
-    }
-
-    public Space getSpace() {
-        return space;
-    }
-
-    public Integer getVolumeIndex() {
-        return volumeIndex;
-    }
-
-    public int getSheetIndex() {
-        return sheetIndex;
+        this.blockLineLocation = blockLineLocation;
```

Comment:
Ok it is here... at the end. Syntactically correct but usually you put things in the same order as the parameters

Reply:
Really?